frontend/smart-kickers-game/src/apis/Game.js

```diff
@@ -0,0 +1,17 @@
+import axios from 'axios';
+
+export async function resetGame() {
+  try {
+    let result = await axios.put(`http://localhost:3000/reset`);
+
+    return {
+      status: result.status,
+      data: result.data,
+    };
+  } catch (e) {
+    return {
+      error: e.response.data,
+      status: e.response.status,
+    };
+  }
```

Comment:
There's no point of returning it if it's not handled.

Reply:
As mentioned in #65 we should ahndle errors so I'm. going to implement it